Description: Tweak build system
 Example programs are useless as far as Debian packages go; since we're
 refreshing the build system anyway, might as well shave a couple of calls
 to GCC in order to speed up compilation a little.
 .
 The ChangeLog must not be regenerated because otherwise building the
 package from a clone of the package repository would overwrite the one
 shipped in the tarball.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2012-05-06
Index: cattle-1.0.git/Makefile.am
===================================================================
--- cattle-1.0.git.orig/Makefile.am	2012-05-06 18:49:02.243098936 +0200
+++ cattle-1.0.git/Makefile.am	2012-05-06 18:50:16.251103111 +0200
@@ -2,7 +2,6 @@
 
 SUBDIRS = \
 	cattle \
-	examples \
 	tests \
 	docs \
 	$(NULL)
@@ -28,33 +27,9 @@
 	$(NULL)
 
 CLEANFILES = \
-	$(srcdir)/ChangeLog \
 	$(NULL)
 
 DISTCHECK_CONFIGURE_FLAGS = \
 	--enable-instrospection \
 	--enable-gtk-doc \
 	$(NULL)
-
-distclean-local:
-	if test "$(srcdir)" = "."; then \
-	  :; \
-	else \
-	  rm -f ChangeLog; \
-	fi
-
-ChangeLog:
-	@echo Creating $@
-	@if test -d "$(srcdir)/.git"; then \
-	  (GIT_DIR=$(top_srcdir)/.git $(top_srcdir)/missing --run git log --stat) > $@.tmp \
-	  && mv -f $@.tmp $@ \
-	  || ($(RM) $@.tmp; \
-	  echo Failed to generate ChangeLog, your ChangeLog may be outdated >&2; \
-	  (test -f $@ || echo git-log is required to generate this file >> $@)); \
-	else \
-	  test -f $@ || \
-	  (echo A git checkout and git-log is required to generate ChangeLog >&2 && \
-	  echo A git checkout and git-log is required to generate this file >> $@); \
-	fi
-
-.PHONY: ChangeLog
Index: cattle-1.0.git/configure.ac
===================================================================
--- cattle-1.0.git.orig/configure.ac	2012-05-06 18:49:02.247098937 +0200
+++ cattle-1.0.git/configure.ac	2012-05-06 18:49:56.183101976 +0200
@@ -92,7 +92,6 @@
                  docs/Makefile
                  docs/reference/Makefile
                  docs/reference/version.xml
-                 examples/Makefile
                  tests/Makefile])
 
 AC_OUTPUT
